tools/ci_build/github/azure-pipelines/templates/validate-nuget.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  NugetPath: ''
+  PlatformsSupported: ''
+  VerifyNugetSigning: ''
+
+steps:
+    - task: PowerShell@2
+      displayName: 'Download AzCopy (used for download test data script)'
```

Comment:
From line 7 to line line 26, they can be replaced by:
```yaml
    - task: UsePythonVersion@0
      displayName: 'Use Python'
      inputs:
        versionSpec: 3.7
```

Reply:
Sure. I can make that change.

Reply:
Made the change